src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderBookingRepository.java

```diff
@@ -71,4 +72,12 @@ List<OffenderBookingId> findDistinctByActiveTrueAndOffenderNomsIdInAndSentences_
     @Override
     @EntityGraph(type = EntityGraphType.FETCH, value = "booking-with-summary")
     Page<OffenderBooking> findAll(@NotNull Specification<OffenderBooking> filter, @NotNull Pageable pageable);
+
+    @Query("""
+    SELECT new uk.gov.justice.hmpps.prison.api.model.LatestTusedData(sc.tusedCalculatedDate, sc.tusedOverridedDate, sc.comments, o.nomsId)
+    from SentenceCalculation sc
+    inner join sc.offenderBooking.offender o where o.nomsId = :nomsId and coalesce(sc.tusedOverridedDate, sc.tusedCalculatedDate) is not null
+    order by sc.calculationDate desc limit 1
```

Comment:
didn't realise that `limit` was a valid JPQL keyword - what does the SQL translate to?

Reply:
I can't find the reference but I think it's a fairly recent addition